rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/450_ip_prefix.yml

```diff
@@ -5,6 +5,7 @@ setup:
         body:
           settings:
             number_of_replicas: 0
+            number_of_shards: 3
```

Comment:
@nik9000 do you think it is ok to force here a specific number of shards or should we rely on "randomness" not specifying the value? As it is with this PR we might not spot issues for the "one-shard" scenario.

Reply:
I'd just rely on the randomness. You can always reproduce it with the failed hash.